testsuite/model/src/test/resources/log4j.properties

```diff
@@ -32,8 +32,8 @@ log4j.logger.org.keycloak.testsuite=${keycloak.testsuite.logging.level}
 # log4j.logger.org.hibernate=debug
 
 # Enable to view loaded SPI and Providers
-# log4j.logger.org.keycloak.services.DefaultKeycloakSessionFactory=debug
-# log4j.logger.org.keycloak.provider.ProviderManager=debug
+ log4j.logger.org.keycloak.services.DefaultKeycloakSessionFactory=debug
```

Comment:
Is this intentional?

Reply:
Good catch, but yes - in the model tests, knowing the actual configuration of the providers is crucial to determine potential cause of the issues.